atmos_model.F90

```diff
@@ -3072,12 +3332,6 @@ subroutine setup_exportdata(rc)
             ! bottom layer meridional wind (v)
             case('inst_merid_wind_height_lowest')
               call block_data_copy_or_fill(datar82d, DYCORE_data(nb)%coupling%v_bot, zeror8, Atm_block, nb, rc=localrc)
-            ! bottom layer zonal wind (u) from physics
-            case('inst_zonal_wind_height_lowest_from_phys')
-              call block_data_copy_or_fill(datar82d, GFS_data(nb)%Statein%ugrs, 1, zeror8, Atm_block, nb, rc=localrc)
```

Comment:
@junwang-noaa I add them in the past for side-by-side run since there was some difference in the wind components in that time but after discussed with @DeniseWorthen and test again, I see that  I could use the existing wind fields rather than introducing new fields. So, I removed them in this PR. I think no other components or configurations are using these fields.